recipe/build.sh

```diff
@@ -1,11 +1,28 @@
 #!/usr/bin/bash
 
-set -x
+set -ex
 
-cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./config
+cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./build-aux
+
+if [[ "$target_platform" == linux* ]]; then
+  rm -f tests/scripts/functions/wildcard
+elif [[ "$target_platform" == osx* ]]; then
+  rm -f tests/scripts/variables/INCLUDE_DIRS
+fi
+
+if [[ ${target_platform} =~ linux* || ${target_platform} =~ .*aarch64.* || ${target_platform} =~ .*ppc64le.* ]]; then
```

Comment:
I prefer in general the pattern:
abc=
if [. <some conditoon> ]; then
  abc="-my-additional-option"
fi

It. is in my opinion more brief and allows further specialization more easy to read